openvsx-sync.json

```
@@ -1,6 +1,66 @@
 [
+    "donjayamanne.githistory",
+    "vscode.github-authentication",
+    "github.vscode-pull-request-github",
+    "ms-vscode.js-debug",
+    "vscode.typescript-language-features",
+    "bazelbuild.vscode-bazel",
+    "octref.vetur",
+    "vscodevim.vim",
+    "errata-ai.vale-server",
+    "llvm-vs-code-extensions.vscode-clangd",
+    "eclipse-cdt.cdt-gdb-vscode",
+    "bmewburn.vscode-intelephense-client",
     "ms-python.python",
+    "asciidoctor.asciidoctor-vscode",
+    "eamodio.gitlens",
+    "golang.go",
+    "dart-code.dart-code",
+    "dart-code.flutter",
     "redhat.vscode-yaml",
+    "ms-kubernetes-tools.vscode-kubernetes-tools",
+    "redhat.vscode-camelk",
     "redhat.java",
-    "redhat.vscode-xml"
+    "vscjava.vscode-java-debug",
+    "vscjava.vscode-java-test",
+    "redhat.vscode-microprofile",
+    "redhat.vscode-quarkus",
+    "redhat.fabric8-analytics",
+    "redhat.vscode-tekton-pipelines",
+    "redhat.project-initializer",
```

```suggestion
    "redhat.project-initializer",
    "redhat.vscode-redhat-account",
```
**redhat.vscode-openshift-connector** depends on **redhat.vscode-redhat-account**